code/game/machinery/cryo.dm

```diff
@@ -12,53 +12,57 @@
 
 	var/on = FALSE
 	var/auto_release = FALSE
+	var/release_notice = FALSE
 	use_power = 1
 	idle_power_usage = 20
 	active_power_usage = 200
+	var/idle_ticks_until_shutdown = 60 //Number of ticks permitted to elapse without a patient before the cryotube shuts itself off to save processing
 
-	var/obj/item/device/radio/cryo_announce
 	var/mob/living/carbon/occupant = null
 	var/obj/item/reagent_container/glass/beaker = null
 
 /obj/machinery/cryo_cell/New()
 
-	cryo_announce = new /obj/item/device/radio(src) //Configure announcer.
-
-	. = ..()
+	return ..()
 
 /obj/machinery/cryo_cell/Dispose()
 	stop_processing()
-	cdel(cryo_announce)
-	cryo_announce = null
-	. = ..()
+	return ..()
 
 /obj/machinery/cryo_cell/examine(mob/living/user)
 	..()
 	if(occupant) //Allows us to reference medical files/scan reports for cryo via examination.
-		if(ishuman(occupant))
-			if(hasHUD(user,"medical"))
-				var/mob/living/carbon/human/H = occupant
-				for(var/datum/data/record/R in data_core.medical)
-					if (R.fields["name"] == H.real_name)
-						if(!(R.fields["last_scan_time"]))
-							to_chat(user, "<span class = 'deptradio'>No scan report on record</span>\n")
-						else
-							to_chat(user, "<span class = 'deptradio'><a href='?src=\ref[src];scanreport=1'>[occupant]: Scan from [R.fields["last_scan_time"]]</a></span>\n")
-						break
+		if(!ishuman(occupant))
+			return
+		if(!hasHUD(user,"medical"))
+			return
+		var/mob/living/carbon/human/H = occupant
+		for(var/datum/data/record/R in data_core.medical)
+			if (!R.fields["name"] == H.real_name)
+				return
```

Comment:
This is changing the logic, not entirely sure what you are after.
`return` will end the proc.
`break` will end the loop.
`continue` will end the current element of the loop and pass on to the next.